app/uk/gov/hmrc/agentoverseasfrontend/views/application/your_tax_registration_numbers.scala.html

```diff
@@ -53,7 +53,7 @@
 
 @editTrnForm(trn: String) = {
     @form(action = routes.TaxRegController.submitUpdateTaxRegNumber, 'id -> s"""updateTrn$trn""") {
-        <input class="hidden" name="original" type="text" id="actual-trn" value="@trn" />
```

Comment:
The only reason I could guess of it being needed was if some test used it to check the element in the output page. I had a look and didn't see any such usage. I've also done a string search through the repository.
I can't think of anything else. Can you?
In fact now that I think about it, I don't think we even need the "updateTrn[x]" id on the form as a whole (line 55)...

Reply:
Fair enough! Was just checking...

It's weird seeing an input without one (used for hints and labels normally, but this is a hidden input so no point in either) and that's only because this is 'hidden input' with the form is basically a link

I guess you're right and we wouldn't need `updateTrn$trn` either if it's not being tested for. but it's not doing any harm... I'll merge it